source/extensions/transport_sockets/tls/cert_validator/spiffe/spiffe_validator.cc

```diff
@@ -171,6 +175,16 @@ int SPIFFEValidator::doVerifyCertChain(X509_STORE_CTX* store_ctx,
   return ret;
 }
 
+int SPIFFEValidator::ignoreCertificateExpirationCallback(int ok, X509_STORE_CTX* store_ctx) {
```

Comment:
If this is just duplicated code from https://github.com/envoyproxy/envoy/blob/50e81276fd8f109ba3a6134e790f65c1cc5bdec9/source/extensions/transport_sockets/tls/cert_validator/default_validator.cc#L230
is it possible to just share the code?

Reply:
done!